public/index.html

```diff
@@ -163,19 +192,61 @@
               />
             </svg>
           </div>
-          <div class="preloadhtml-header-link-item">Campaign</div>
+          <div class="preloadhtml-header-link-item">Analytics</div>
           <div class="preloadhtml-header-link-item">About</div>
         </div>
       </div>
       <div style="height: 180px; display: flex;">
         <img
-          style="animation: pulse 800ms linear infinite; margin: auto; "
+          class="onlyVisibleInDarkMode"
+          style="animation: pulse 800ms linear infinite; margin: auto;"
           width="160px"
           src="/logo-dark.svg"
           alt="DmmLogo"
         />
+        <img
+          class="onlyVisibleInLightMode"
+          style="animation: pulse 800ms linear infinite; margin: auto;"
+          width="160px"
+          src="/logo.svg"
+          alt="DmmLogo"
+        />
       </div>
     </div>
+    <script>
+      ;(() => {
```

Comment:
`;` redundant

Reply:
It's auto added

Reply:
Maybe this is the answer: https://stackoverflow.com/questions/17978883/what-is-the-purpose-of-a-semicolon-before-an-iife

Reply:
cool